README.md

```diff
@@ -145,6 +145,10 @@ CAT_OAUTH_RESTRICT_ACCESS=org:my-organization
 CAT_OAUTH_RESTRICT_REQUESTER_ACCESS=team:123456
 ```
 
+## Deployment
+
+This site is deployed in `production` and in `staging` following our [web application deployment workflow](https://github.com/src-d/guide/blob/master/engineering/continuous-delivery.md)
```

Comment:
This is something important - we should think (and document it as well) that it's not a web site or a service, it's an application. Ofcourse as a web application, it can be deployed by different people in different scenarios, but I think it's worth highlighting it here.

How about 
 - changing the title from `Deployment` -> `source{d} internal deployment`
 - changing `site` -> `application`
 - `production and staging  sourced{d} environments`

Reply:
I just copied the code from Blog and Landing but yes: your proposals are even better and they could be also applied there.
I'll proceed with your suggestions before merging, Thanks!